.travis.yml

```diff
@@ -32,13 +32,27 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install ninja python3; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then docker pull jpakkane/mesonci:zesty; fi
+  - pip3 install codecov
```

Comment:
It should be run only on OSX then. The Linux container should have everything by default.